app/views/prior_authority/applications/_drafts.html.erb

```diff
@@ -40,8 +40,16 @@
             <%= application.laa_reference %>
           </td>
           <td class="govuk-table__cell">
-            <%= link_to t("prior_authority.applications.tabs.delete"),
-                        confirm_delete_prior_authority_application_path(application) %>
+            <a
+              href="<%=confirm_delete_prior_authority_application_path(application)%>"
+              class="govuk-link--no-visited-state"
+            >
+              <%= t("prior_authority.applications.tabs.delete") %>
+              <span class="govuk-visually-hidden">
+                applications
```

Comment:
I think this should be `application` rather than `applications` as there's only one of them, and strictly this should be I18nised as `<%= t('.application') %>`

Reply:
Oh applications is what was on the ticket so I just copied that text, but you're right